R/FilterState.R

```diff
@@ -530,6 +569,23 @@ FilterState <- R6::R6Class( # nolint
         }
       str2lang(ans)
     },
+    get_metadata_id = function() {
+      private$metadata_id
```

Comment:
To make it consistent with `get_metadata_title`:

```suggestion
      if (identical(private$metadata_id, character(0))) {
        private$get_varname()
      } else {
        private$metadata_id
      }
```